host/tests/dsCompositeIn_L3_Tests/dsCompositeIn_test3_ScaleAndVerify_Video.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../"))
+
+from dsCompositeInHelperClass import dsCompositeInHelperClass
+
+class dsCompositeIn_test3_ScaleAndVerify_Video(dsCompositeInHelperClass):
+    """
+    Test class to scale and verify the video in CompositeIn device
+
+    This class uses the `dscompositeInClass` to interact with the device's compositeIn ports,
+    downloading necessary test assets, selecting
+    CompositeIn ports, and performing verification of compositeIn output.
+    """
+
+    def __init__(self):
+        """
+        Initializes the test3_ScaleAndVerify_Video test .
+
+        Args:
+            None.
+        """
+        self.testName  = "test3_ScaleAndVerify_Video"
+        super().__init__(self.testName, '1')
```

Comment:
here test number should be 3. Can you check all over the tests

Reply:
done